zeek/manifest.json

```
@@ -0,0 +1,98 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "81ba5f4a-0e85-48c3-9ba3-2e5ea37b1ed2",
+  "app_id": "zeek",
+  "display_on_public_website": true,
```

```suggestion
  "display_on_public_website": false,
```

We will set this to `true` once Agent 7.52.0 will be released